sdk/sdk-skeleton.go

```diff
@@ -0,0 +1,77 @@
+type Asset struct{}
+
+//// Methods directly on the SDK
+
+func NewChannel() {
+
+}
+
+func NewChannelListenerService() {
+
+}
+
+//// Channel State Machine
+
+type ChannelStateMachine struct {
+	Status                 string // waiting | active | closing | closed
+	Asset                  Asset
+	InitiatorEscrowAccount string
+	ResponderEscrowAccount string
+	InitiatorClaimAmount   int
+	ResponderClaimAmount   int
+	IsInitiator            bool
+	MyAccount              string
+	MyClaimAmount          int
+	OtherAccount           string
+	OtherClaimAmount       int
+}
+
+func (c *ChannelStateMachine) Init() {
+	return nil
+}
+
+func (c *ChannelStateMachine) CheckChannel() (*ChannelCheckResponse, error) {
+	return nil, error
+}
+
+func (c *ChannelStateMachine) Payment(sendAmount int, timeoutMillis int) error {
```

Comment:
Let's deal with timeouts in another way. We will probably add context's to all the functions that need it. We can think about that later though.
```suggestion
func (c *ChannelStateMachine) Payment(sendAmount int) error {
```

Reply:
https://github.com/stellar/experimental-payment-channels/pull/25/commits/462d4215919a3528d809eed806adc7360e4f9534